x-pack/plugins/apm/public/components/app/TransactionDetails/Transaction/StickyTransactionProperties.tsx

```diff
@@ -91,5 +95,28 @@ export function StickyTransactionProperties({
     }
   ];
 
+  if (errorCount) {
+    stickyProperties.push({
+      label: i18n.translate('xpack.apm.transactionDetails.errorsLabel', {
+        defaultMessage: 'Errors'
+      }),
+      val: (
+        <KibanaLink
+          pathname={'/app/apm'}
+          hash={`/${idx(transaction, _ => _.service.name)}/errors`}
+          query={{
+            kuery: `transaction.id~3A~22${transaction.transaction.id}~22`
```

Comment:
Use `legacyEncodeURIComponent`:
```suggestion
            kuery: legacyEncodeURIComponent(`transaction.id : "${transaction.transaction.id}"`)
```